### DISCUSSION TITLE
What is the difference between this and Obsidian Publisher?

### DISCUSSION THREAD

**Discussion by ooker777** [GooRoo/mkdocs-obsidian-bridge#9] [General]
This plugin also exports your vault to MkDocs https://obsidian-publisher.netlify.app/

**Comment by GooRoo** [comment]
As far as I can tell, my plugin is kinda what @Lisandra-dev achieved in [Lisandra-dev/mkdocs-ezlinked-plugin](https://github.com/Lisandra-dev/mkdocs-ezlinked-plugin) for her [ObsidianPublisher/obsidian-github-publisher](https://github.com/ObsidianPublisher/obsidian-github-publisher).

In general, [Obsidian Publisher](https://github.com/ObsidianPublisher/obsidian-github-publisher) provides more holistic solution to the same problem. However, she chose an aproach which I personally find a bit overcomplicated. My initial idea was to make it possible to use an Obsidian Vault as a source for MkDocs without any modifications or installations of additional Obsidian plugins—“as is”. 

I suggest you to choose whatever fits you best 🙂

**Reply by Mara-Li** [reply]
My goal was to create a plugin that fit everyone needs! So it was linked to mkdocs in start but was extended rapidly. It can be use with everything that supports md file umu

Also in the start in don't want to share all my file + have all on github (private...)

**Reply by ooker777** [reply]
@GooRoo in that case, why don't you contribute to the plugin to make it bug-free?

**Reply by GooRoo** [reply]
@ooker777 And why don't you?

**Reply by ooker777** [reply]
I do report bugs. What I mean is that the ezlinked plugin is the closest plugin to your need. Isn't that improving an existing one less work than building from scratch?

**Reply by GooRoo** [reply]
There are pros and cons for either way as well as some nuances.

On the one hand, my plugin does what I need with multiple times less code. And less code is usually better because the complexity level is lower, so _potentially_ it contains less bugs.

On the other hand, discoverability is still a problem. When I was looking for what I need, I didn't find a good alternative that would fit my needs.

Additionally, competitiveness is generally a good thing since it drives the development forward. If my plugin does the job worse, then it won't be used by as many people as alternatives (and actually this is exactly the case as we see). Or it may mean that I haven't spent enough time for advertisement (which is also true).

But if you still insist that I'd better spend my time for something else rather than for my own plugin, then just donate me money, and we'll see what I can do 😉

**Reply by ooker777** [reply]
I don't insist, I just want to understand. So if I understand you correctly, you concluded that to get the plugin to work as it should, the amount of work for refactoring is large enough to be nearly the same with the amount of work for rewriting it from the start. And since you see that your needs are different to her needs, writing a new one is your best option. Is that correct?

**Reply by GooRoo** [reply]
Kinda like that, yes. Maybe I'll reconsider this in the future.
